themes/bootstrap3/templates/search/results-scripts.phtml

```diff
@@ -11,3 +11,9 @@ if (($this->listViewOption ?? 'full') !== 'full') {
     $this->headScript()->appendFile('record.js');
     $this->headScript()->appendFile('embedded_record.js');
 }
+if ($this->jsResults ?? false) {
+    $this->headScript()->appendFile('search.js');
+    // Ensure OpenURL support is available even if none of the results on the initial
+    // page has one:
+    $this->headScript()->appendFile('openurl.js');
```

Comment:
Would a possible compromise on the openurl.js situation be to move this outside of the conditional block and then remove it from being included in other places? Or do we have to worry about OpenURL embedding outside of search results? If that is the case, then I'd support moving it into the theme config for global activation both to simplify things and to increase the benefit of using the asset pipeline.

Reply:
We need to support OpenURL embedding outside of search results, like in favorite lists as well. I'll move it to theme config.